code/modules/mob/living/simple_animal/abnormality/_tools/he/shelter.dm

```diff
@@ -154,6 +154,9 @@
 	. = ..()
 	if(!ishostile(M)) // kick out any hostile mobs
 		return
+	var/mob/living/simple_animal/hostile/G = M
+	if(("neutral" in G.faction) && ishostile(G)) //This should prevent an exploit with clerkbots and any other friendly summon teleporting
```

Comment:
```suggestion
	if("neutral" in G.faction) //This should prevent an exploit with clerkbots and any other friendly summon teleporting
```
Hostile check is redundant, we wouldn't have gotten this far without them being hostile due to line 155